content/rest-api/artifacts.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Artifacts API
+description: Authenticated and public access to build artifacts
+weight: 3
+---
+
+## Authenticated download URL
+
+`GET /artifacts/:uuid1/:uuid2/:artifactName`
+
+This URL can be obtained using the [Builds API](/rest-api/builds) or copied directly from the Codemagic UI.
```

Comment:
Should we have an example for this as well?

Reply:
It's just a URL to copy-paste in browser to download artifact right away, no `curl` requests required